index.js

```diff
@@ -35,10 +35,8 @@ var FeatureService = function (url, options) {
   this.timeOut = 1.5 * 60 * 1000
   var concurrency = this.url.split('//')[1].match(/^service/) ? 16 : 4
 
-  // an async for requesting pages of data
-  this.pageQueue = queue(function (task, callback) {
-    this._requestFeatures(task, callback)
-  }.bind(this), concurrency)
```

Comment:
Thanks for catching that. I wonder if `this.pageQueue = queue(this._requestFeatures.bind(this), concurrency)` would work. That would still remove the unnecessary closure while preserving context.